src/modules/0.0.22/aws_codepipeline/mappers/supported_regions.ts

```diff
@@ -0,0 +1,22 @@
+// Manually extracted from https://docs.aws.amazon.com/general/latest/gr/codepipeline.html
+// as there is no API for this :(
+const supportedRegions = [
+  'ap-east-1',
+  'ap-northeast-1',
+  'ap-northeast-2',
+  'ap-south-1',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ca-central-1',
+  'eu-central-1',
+  'eu-south-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-west-3',
+  'sa-east-1',
+  'us-east-1',
+  'us-east-2',
+  'us-west-1',
+  'us-west-2',
+];
+export default supportedRegions;
```

Comment:
I might have missed this, but why didn't we go multi-region from the get go?

Reply:
no, i could not add it because of ec2 dependency, that is not multi-region yet